doc/release-notes/6.2-release-notes.md

```diff
@@ -0,0 +1,401 @@
+
+### Return to Author Now Requires a Reason
+
+The Popup for returning to author now allows to type in a message to explain the reasons of return and potential edits needed, that will be sent by email to the author.
+
+Please note that this note is mandatory, but that you can still type a creative and meaningful comment such as "The author would like to modify his dataset", "Files are missing", "Nothing to report" or "A curation report with comments and suggestions/instructions will follow in another email" that suits your situation.
+
+<a id="multiple-pid-sup"></a>
+### Support for Using Multiple PID Providers
+
+This release adds support for using multiple PID (DOI, Handle, PermaLink) providers, multiple PID provider accounts
+(managing a given protocol, authority,separator, shoulder combination), assigning PID provider accounts to specific collections,
+and supporting transferred PIDs (where a PID is managed by an account when its authority, separator, and/or shoulder don't match
+the combination where the account can mint new PIDs). It also adds the ability for additional provider services beyond the existing
+DataCite, EZId, Handle, and PermaLink providers to be dynamically added as separate jar files.
+
+These changes require per-provider settings rather than the global PID settings previously supported. While backward compatibility 
+for installations using a single PID Provider account is provided, updating to use the new microprofile settings is highly recommended
+and will be required in a future version.
+
+[New microprofile settings](#microprofile-settings)
+
+
+### Rate Limiting Using JCache (With Hazelcast As Provided by Payara)
+
+The option to rate limit has been added to prevent users from over taxing the system either deliberately or by runaway automated processes.
+Rate limiting can be configured on a tier level with tier 0 being reserved for guest users and tiers 1-any for authenticated users.
+Superuser accounts are exempt from rate limiting.
+Rate limits can be imposed on command APIs by configuring the tier, the command, and the hourly limit in the database.
+Two database settings configure the rate limiting.
+Note: If either of these settings exist in the database rate limiting will be enabled.
+If neither setting exists rate limiting is disabled.
+
+`:RateLimitingDefaultCapacityTiers` is a comma separated list of default values for each tier.
+In the following example, the default for tier `0` (guest users) is set to 10,000 calls per command per hour and tier `1` (authenticated users) is set to 20,000 calls per command per hour.
+Tiers not specified in this setting will default to `-1` (No Limit). I.e., -d "10000" is equivalent to -d "10000,-1,-1,..."
+
+For more details check the detailed guide on [this link](https://guides.dataverse.org/en/6.2/installation/config.html#rate-limiting).
+
+Hazelcast is configured in Payara and should not need any changes for this feature
```

Comment:
```suggestion
For more information check the detailed guide at [this link](https://guides.dataverse.org/en/6.2/installation/config.html#rate-limiting).
```

I think this is enough. However, let's make sure both new database settings are mentioned in the release notes.